src/assets/templates/typescript/src/steps/users/user-field-equals.ts

```diff
@@ -60,25 +69,35 @@ export class UserFieldEqualsStep extends BaseStep implements StepInterface {
       return this.error('There was a problem connecting to JSON Placeholder.');
     }
 
-    if (apiRes.body.length === 0) {
-      // If no results were found, return an error.
-      return this.error('No user found for email %s', [email]);
-    } else if (!apiRes.body[0].hasOwnProperty(field)) {
-      // If the given field does not exist on the user, return an error.
-      return this.error('The %s field does not exist on user %s', [field, email]);
-    } else if (apiRes.body[0][field] == expectedValue) {
-      // If the value of the field matches expectations, pass.
-      return this.pass('The %s field was set to %s, as expected', [
-        field,
-        apiRes.body[0][field],
-      ]);
-    } else {
-      // If the value of the field does not match expectations, fail.
-      return this.fail('Expected %s field to be %s, but it was actually %s', [
-        field,
-        expectedValue,
-        apiRes.body[0][field],
-      ]);
+    try {
+      if (apiRes.body.length === 0) {
+        // If no results were found, return an error.
+        return this.error('No user found for email %s', [email]);
+      } else if (!apiRes.body[0].hasOwnProperty(field)) {
+        // If the given field does not exist on the user, return an error.
+        return this.error('The %s field does not exist on user %s', [field, email]);
+      } else if (this.compare(operator, apiRes.body[0][field], expectedValue)) {
+        // If the value of the field matches expectations, pass.
+        return this.pass(this.operatorSuccessMessages[operator], [
```

Comment:
Now that I think about it, do you think there's a need to add `operatorSuccessMessages` and `operatorFailMessages` to the base step?  Since we have to `import * from @run-crank/utilities` anyway, we could just reference `util.operatorSuccessMessages`

What do you think @amcpanaligan?

Reply:
🤔

Reply:
Definitely yes! I remember the reason why it's not changed to directly access `util`. So that we won't change all the `base step` in each cogs. But for the template we can move on with the `util` call.

Reply:
I included this in last commit